sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4693,10 +4761,12 @@ public partial class RollingUpgradePolicy
         public RollingUpgradePolicy() { }
         public bool? EnableCrossZoneUpgrade { get { throw null; } set { } }
         public int? MaxBatchInstancePercent { get { throw null; } set { } }
+        public bool? MaxSurge { get { throw null; } set { } }
```

Comment:
does it make sense if we could rename this to `IsMaxSurge`? Is this the intention of this property?